gradle-android-command-plugin/src/test/groovy/com/novoda/gradle/command/AndroidCommandPluginExtensionTest.groovy

```diff
@@ -15,8 +15,10 @@ class AndroidCommandPluginExtensionTest extends GroovyTestCase {
     }
 
     private AndroidCommandPluginExtension createExtension() {
-        def project = ProjectBuilder.builder().build()
+	def projectDir = new File('..')
+        def project = ProjectBuilder.builder().withProjectDir(projectDir).build()
```

Comment:
Hmm, don't know. Any ideas? 
This just fixes the breaking test. 
The current setup is not really valuable, we need to reconsider that. May be set up a project just for testing?